TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Windows Terminal Profile Settings
+description: Learn how to customize the individual profiles within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: overview
+ms.service: terminal
+---
+
+# Profile settings in the Windows Terminal
+
+The properties listed below are specific to each unique profile.
+
+## Unique identifier
+
+Profiles can use a GUID as a unique identifier. To make a profile your default profile, it needs a GUID for the `defaultProfile` global setting.
+
+**Property name:** `guid`
+
+**Necessity:** Required
+
+**Accepts:** GUID as a string in registry format: `"{00000000-0000-0000-0000-000000000000}"`
+
+<br />
+
+___
+
+## Executable settings
+
+### Command line
+
+The executable used in the profile.
+
+**Property name:** `commandline`
+
+**Necessity:** Optional
+
+**Accepts:** Executable file name as a string
+
+### Source
+
+Stores the name of the profile generator that originated this profile. _There are no discoverable values for this field._
+
+**Property name:** `source`
+
+**Necessity:** Optional
+
+**Accepts:** String
+
+### Starting directory
+
+The directory the shell starts in when it is loaded.
+
+**Property name:** `startingDirectory`
+
+**Necessity:** Optional
+
+**Accepts:** Folder location as a string
+
+**Default value:** `"%USERPROFILE%"`
+
+<br />
+
+___
+
+## Dropdown settings
+
+### Name
+
+This is the name of the profile that will be displayed in the dropdown menu. This value is also used as the "title" to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application. This "title" behavior can be overridden by using `tabTitle`.
+
+**Property name:** `name`
+
+**Necessity:** Required
+
+**Accepts:** String
+
+### Icon
+
+Set the icon that displays within the tab and the dropdown menu.
+
+**Property name:** `icon`
+
+**Necessity:** Optional
+
+**Accepts:** File location as a string
+
+### Hide a profile from the dropdown
+
+If `"hidden"` is set to `true`, the profile will not appear in the list of profiles. This can be used to hide default profiles and dynamically generated profiles, while leaving them in your settings file.
+
+**Property name:** `hidden`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `false`
+
+<br />
+
+___
+
+## Tab title settings
+
+### Custom tab title
```

Comment:
Definitely going to need gifs in this area (and the few below and above). I'm having a hard time visualizing what the difference between setting this one is and the one a few blocks up.